tests/test_wsireader.py

```diff
@@ -161,6 +161,22 @@
     torch.tensor([[[242], [242]], [[242], [242]], [[242], [242]]], dtype=torch.float32),
 ]
 
+TEST_CASE_10_MPP = [
+    FILE_PATH,
+    {},
+    {"location": (HEIGHT // 2, WIDTH // 2), "size": (2, 1), "mpp": 1000},
```

Comment:
Could we add more tests about the other two ways?

Reply:
Do you mean for `power`?
The current WSI that we are using for testing does not include metadata for `power` so I am trying to find a no-very-large WSI that has it to add the unittests.

Reply:
do you mean setting it in the object instantiation?

Reply:
I mean set `power` here.

Reply:
I added several unittests to cover power, mpp, and their variations.